termine-be/cli.py

```diff
@@ -342,21 +343,66 @@ def get_bookings_created_at(db: directives.PeeweeSession, booked_at: hug.types.t
 
 
 @hug.cli(output=hug.output_format.pretty_json)
-def free_slots_at(db: directives.PeeweeSession, user: hug.types.text, at_datetime: hug.types.text = None):
+def free_slots_at(db: directives.PeeweeSession, user: hug.types.text, at_datetime: hug.types.text = None, max_days_after: hug.types.number = 2):
     """
     args: USER_NAME [--at_datetime ISO_DATETIME]
```

Comment:
Dokumentation veraltet schneller als Code :)

Reply:
(Der wird beim Aufruf von hug -f cli.py o.ä. aufgelistet)

Reply:
```
root@fd924520548a:/app# hug -f main.py -c free_slots_at
usage: hug [-h] [-a AT_DATETIME] [-m MAX_DAYS_AFTER] user
hug: error: the following arguments are required: user
```

scheint mir so, als hätte der Kommentar keinerlei Auswirkung, aber guter catch. Ich lösche den Kommentar.

Reply:
Die Kommentare werden in der Übersicht der cli commands mit angezeigt. (Nur zur Info, is mir gleich obs drin bleibt)

Reply:
`hug -f main.py -c cli`

danke, den kannte ich noch nicht, die kommentare kommen wieder rein!